app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,71 @@
+class TweetsController < ApplicationController
+
+get '/tweets' do
+  if logged_in?
+    @tweets = Tweet.all
+    erb :'/tweets/tweets'
+  else
+    redirect to '/login'
+  end
+end
+
+get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if !params['content'].empty?
+      @tweet = Tweet.create(:content => params['content'], :user_id => current_user.id)
+      @tweet.save
+      redirect to "/tweets/#{@tweet.id}"
+    else
+      redirect to "/tweets/new"
+    end
+  end
+
+  get '/tweets/:id' do
+    if logged_in?
+      @tweet = Tweet.find(params[:id])
+      erb :'/tweets/show_tweets'
+    else
+      redirect to "/login"
+    end
+  end
+
+  get '/tweets/:id/edit' do
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?